doc/source/user_guide/documentation/variables.rst

```diff
@@ -13,6 +13,17 @@ Constructor
    Variables
 
 
+Attributes
+----------
+
+.. autosummary::
+   :toctree: ../../_icepyx/
+
+   Variables.path
+   Variables.product
+   Variables.version
```

Comment:
Could you add docstrings for these three attributes, so that there's a description of them? Right now it shows up empty under https://icepyx--657.org.readthedocs.build/en/657/user_guide/documentation/variables.html

![image](https://github.com/user-attachments/assets/9bfe1c56-64f4-4499-994e-88f776869d8d)

Docstrings can be added here:

https://github.com/icesat2py/icepyx/blob/0512d75e73a19b5a9f2ef79e52494d8e181a0c85/icepyx/core/variables.py#L97-L107

and you can copy the description from here:

https://github.com/icesat2py/icepyx/blob/0512d75e73a19b5a9f2ef79e52494d8e181a0c85/icepyx/core/variables.py#L32-L40